env.sample

```diff
@@ -1,34 +1,111 @@
 # Copy this file to .env and replace the credentials with
 # your own before starting the app.
 
-# Watson Assistant
-ASSISTANT_APIKEY=<add_assistant_apikey>
-ASSISTANT_URL=<add_assistant_url>
+#----------------------------------------------------------
+# IBM Cloud
+#
+# If your services are running on IBM Cloud,
+# uncomment and configure these.
+# Remove or comment out the IBM Cloud Pak for Data sections.
+#----------------------------------------------------------
 
-# Optionally, use a non-default skill by specifying your own workspace ID or name.
+# Watson Assistant
+# CONVERSATION_AUTH_TYPE=iam
```

Comment:
I consider it a regression to go from ASSISTANT back to CONVERSATION.
Simple workaround is here (Python example):  https://github.com/IBM/watson-voice-bot/blob/master/app.py#L125